### PR TITLE
98 Stellar implementation

### LINKED ISSUE #98: Add the Stellar implementation
ℹ️ This issue is part of the Epic #95

### Describe the solution you'd like

Add the Stellar implementation and the mocks for the transaction functions.

### Describe alternatives you've considered

Add the Stellar implementation and the mocks for the functions, create account, create asset, authorize asset, and process payment..

### PR DESCRIPTION
**Pull request by miguelnietoa**
## Description
This PR just create the boundaries for Stellar and the Stellar implementation itsself using the Crypto behaviour.

Fixes #98 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/mintacoin/lib/mintacoin/crypto/stellar/assets.ex [hunk 1]
```diff
@@ -0,0 +1,70 @@
+defmodule Mintacoin.Crypto.Stellar.Assets do
+  @moduledoc """
+  This module provides functions for creating and managing assets in the Stellar network.
+  """
+
+  @spec create(params :: map()) :: {:ok, struct()}
+  def create(_params) do
+    {:ok,
+     %Stellar.Horizon.Transaction{
```
Comment (juanhurtado10) [inline apps/mintacoin/lib/mintacoin/crypto/stellar/assets.ex:9]: @miguelnietoa @CristhianRodriguezMolina We should return here a Mintacoin struct, please do not forget the system knows nothing about internal implementations or blockchain's responses.

**Review by juanhurtado10 — CHANGES_REQUESTED** [review]

File: apps/mintacoin/lib/mintacoin/crypto/stellar/accounts.ex [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  @spec create(params :: map()) :: {:ok, TxResponse.t()}
+  def create(_params) do
+    {:ok,
+     %TxResponse{
+       successful: true,
+       hash: "7f82fe6ac195e7674f7bdf7a3416683ffd55c8414978c70bf4da08ac64fea129",
+       created_at: ~U[2022-06-29 15:45:45Z],
+       blockchain: :stellar,
+       tx_json: %{
```
Comment (juanhurtado10) [inline apps/mintacoin/lib/mintacoin/crypto/stellar/accounts.ex:16]: @miguelnietoa @CristhianRodriguezMolina is this attribute storing a JSON or a map?

**Review by juanhurtado10 — CHANGES_REQUESTED** [review]

File: apps/mintacoin/lib/mintacoin/crypto/stellar/accounts.ex (hunk 2 above)
Comment (CristhianRodriguezMolina) [reply apps/mintacoin/lib/mintacoin/crypto/stellar/accounts.ex:16]: Is storing a JSON, so we can use `Jason` to encode the map into something like this: `"{\"hello\":\"world\"}"`

Comment (juanhurtado10) [reply apps/mintacoin/lib/mintacoin/crypto/stellar/accounts.ex:16]: OK great, please name it `tx_json` if you're planning to store a JSON or `raw_tx` if the content is a map.

Comment (CristhianRodriguezMolina) [reply apps/mintacoin/lib/mintacoin/crypto/stellar/accounts.ex:16]: Ok we will rename it to `raw_tx`.

**Review by juanhurtado10 — APPROVED** [review]

**Pull request merged**